app/views/directives/_edit-command.html

```diff
@@ -25,23 +25,23 @@
         <span as-sortable-item-handle class="input-group-addon action-button drag-handle">
           <i class="fa fa-bars" aria-hidden="true"></i>
         </span>
-        <a href="" ng-click="removeArg($index)" class="input-group-addon action-button remove-arg" title="Remove Argument">
-          <span class="sr-only">Remove Argument</span>
+        <a href="" ng-click="removeArg($index)" class="input-group-addon action-button remove-arg" title="Remove Item">
+          <span class="sr-only">Remove Item</span>
           <i class="pficon pficon-close" aria-hidden="true"></i>
         </a>
       </span>
     </span>
   </p>
   <div class="form-group">
-    <label class="sr-only" ng-attr-for="{{id}}-add-arg">Add Argument</label>
+    <label class="sr-only" ng-attr-for="{{id}}-add-arg">Add {{type || 'Command'}}</label>
     <!-- Single-line entry -->
     <span ng-show="!multiline" class="input-group">
       <input type="text"
              ng-model="nextArg"
              name="nextArg"
              ng-attr-id="{{id}}-add-arg"
              on-enter="addArg()"
-             placeholder="Add argument"
+             placeholder="Add {{type || 'command'}}"
```

Comment:
Changing the default will affect other places where this is used, where it should be "Add argument"